dockers/docker-database/Dockerfile.j2

```diff
@@ -32,6 +32,8 @@ RUN apt-get clean -y                                  && \
              s/^# unixsocket/unixsocket/;                \
              s/redis-server.sock/redis.sock/g;           \
              s/^client-output-buffer-limit pubsub [0-9]+mb [0-9]+mb [0-9]+/client-output-buffer-limit pubsub 0 0 0/; \
+             s/^bind 127.0.0.1 ::1$/# bind 127.0.0.1 ::1/;       \
```

Comment:
># bind 127.0.0.1 [](http://example.com/codeflow?start=36&length=16)

If you comment the bind line, it will listen to all interfaces, which is a security concern.